electron/libelectron/src/backends/memory_mgmt.hh

```diff
@@ -0,0 +1,30 @@
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+#include <assert.h>
+
+namespace Memory_mgmt{
+
+class Memory_mgmt{
```

Comment:
As you decouple the device memory ownership from this class, I'd recommend renaming this appropriately. My suggestion: `FirstFitAllocator`

Reply:
Done